scripts/dotnet-cli-build/PublishTargets.cs

```diff
@@ -118,6 +116,41 @@ public static BuildTargetResult PublishDebFileToDebianRepo(BuildTargetContext c)
             return c.Success();
         }
 
+        [Target]
+        [Environment("DOCKER_HUB_REPO")]
+        [Environment("DOCKER_HUB_TRIGGER_TOKEN")]
+        public static BuildTargetResult TriggerDockerHubBuilds(BuildTargetContext c)
+        {
+            string dockerHubRepo = Environment.GetEnvironmentVariable("DOCKER_HUB_REPO");
+            string dockerHubTriggerToken = Environment.GetEnvironmentVariable("DOCKER_HUB_TRIGGER_TOKEN");
+
+            Uri baseDockerHubUri = new Uri("https://registry.hub.docker.com/u/");
+            Uri dockerHubTriggerUri;
+            if (!Uri.TryCreate(baseDockerHubUri, string.Format("{0}/trigger/{1}/", dockerHubRepo, dockerHubTriggerToken), out dockerHubTriggerUri))
+            {
+                return c.Failed("Invalid DOCKER_HUB_REPO and/or DOCKER_HUB_TRIGGER_TOKEN");
+            }
+
+            c.Info("Triggering automated DockerHub builds for " + dockerHubRepo);
+            using (HttpClient client = new HttpClient())
+            {
+                StringContent requestContent = new StringContent("{\"build\": true}", Encoding.UTF8, "application/json");
+                try
+                {
+                    HttpResponseMessage response = client.PostAsync(dockerHubTriggerUri, requestContent).Result;
+                    if (!response.IsSuccessStatusCode)
+                    {
+                        return c.Failed(string.Format("HTTP request to {0} was unsuccessful. HTTP {1}: {2}", dockerHubTriggerUri.ToString(), response.StatusCode, response.ReasonPhrase));
```

Comment:
Consider writing the response's Content here as well, as that will give you better error messages when things go wrong.